<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsStr2str</class>
 <widget class="QDialog" name="OptionsStr2str">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="InStreamlineEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>251</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>BASE STATION CONFIG</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDefault1">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>310</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>START BASE</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>731</width>
     <height>241</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>BASE POSITION</string>
    </attribute>
    <widget class="QRadioButton" name="radioButtonPositionAuto">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>AU&amp;TOMATIC </string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonPositionManual">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>20</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>&amp;MANUAL INPUT</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonManual">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>90</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>WRITE</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="LatManualLine">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>60</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>01.00</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>100</y>
       <width>101</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>      LONGITUDE :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>140</y>
       <width>101</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>ALTITUDE :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>60</y>
       <width>101</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>LATITUDE :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="LongManualLine">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>100</y>
       <width>111</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>01,00</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="AltiManualLine">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>140</y>
       <width>111</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>1.00</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>101</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>LATITUDE :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_13">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>140</y>
       <width>101</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>ALTITUDE :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>101</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>      LONGITUDE :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="LatAutoLine">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>111</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>01,00</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="LongAutoLine">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>01.00</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="AltiAutoLine">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>140</y>
       <width>111</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>1.00</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>CONFIGURATION</string>
    </attribute>
    <widget class="QComboBox" name="OutBaudRatecomboBox">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>100</y>
       <width>81</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>57600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>38400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="OutSerialPortcomboBox">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>100</y>
       <width>141</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>serial://ttyUSB0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>serial://ttyAMA0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>serial://ttyACM0</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="OutFormatcomboBox">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>100</y>
       <width>71</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>rtcm3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>rtcm2</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>70</y>
       <width>141</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>SERIAL PORT</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>70</y>
       <width>81</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>BAUDRATE</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_6">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>70</y>
       <width>71</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>FORMAT</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_7">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>20</y>
       <width>201</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>OUTPUT STREAM</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_8">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>201</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>INPUT STREAM</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="InSerialPortcomboBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>141</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>serial://ttyACM0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>serial://AMA0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>serial://USB0</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="InBaudratecomboBox">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>110</y>
       <width>81</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>57600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>38400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="InFormatcomboBox">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>110</y>
       <width>81</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>ubx</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>nvs</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>oem3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>nov</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ss2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>hemis</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>stq</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>javad</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>binex</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>rtcm2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>rtcm3</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="lineEdit_9">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>80</y>
       <width>81</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>FORMAT</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>141</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>SERIAL PORT</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_11">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>80</y>
       <width>81</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>BAUDRATE</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_15">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>140</y>
       <width>311</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>RTCM MESSAGES</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="RtcmMsgcomboBox">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>170</y>
       <width>311</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>1004,1006,1008,1012,1019,1020</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="UpdateOptionspushButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>160</y>
       <width>271</width>
       <height>34</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>UPDATE STREAM  CONFIGURATION</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QTextBrowser" name="RtkrcvOptionstextBrowser">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>591</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
